### PR TITLE
add a Symbol polyfill

### LINKED ISSUE #5: Internet Explorer 11 compatibility?
Should loglevelnext work with IE11 out of the box? The [browser support](https://github.com/shellscape/loglevelnext#browser-support) section mentions IE 10 as the minimum IE version supported, but I'm running into a JS error when hitting my webpack-serve instance on IE 11 due to the use of `Symbol` in this library. If the intention is to support IE11, would you have any objection to replacing the `Symbol`s?

* Operating System: macOS 10.13.4
* Node Version: 8.11.0
* Browser and Version (if applicable): Internet Explorer 11 (running in a Windows 7 VM)
* NPM Version: 5.6.0
* loglevelnext Version: 1.0.4

- [x] This is a **bug**
- [ ] This is a **feature** request
- [x] This is a **modification** request

### Code
This affects anyone using webpack-hot-client along with Internet Explorer 11 as it chokes on [this line](https://github.com/shellscape/loglevelnext/blob/2a7ab4303beca56fb58ad2baa6a1894704c65275/lib/MethodFactory.js#L4) due to `Symbol` not being supported in Internet Explorer.

### Expected Behavior
Internet Explorer does not throw a "Symbol is undefined" error.

### Actual Behavior
Internet Explorer throws an error
```
'Symbol' is undefined
```

### For Bugs; How can we reproduce the behavior?
Spin up `webpack-serve` (or anything using webpack-hot-client) and access it with Internet Explorer 11. For those of us who aren't on a machine that has Internet Explorer, VMs are available [here](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/).

### PR DESCRIPTION
**Pull request by seanparmelee**
- [x] This is a **bugfix**
- [ ] This is a new **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **typo fix**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

No

### Motivation / Use-Case

To maintain IE 10 & 11 compatibility, `Symbol` needs to be polyfilled since these browsers do not support it. Fixes #5. 

### Breaking Changes

None

### Additional Info

This PR builds off the commit in your ie-assign-symbol branch; not sure if you wanted those changes regardless but if not I can remove them. With this PR, the loglevelnext.min.js file bumps up to 18 kB. I also gave `useBuiltIns: 'usage'` a shot and that bumped the size to 30 kB.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
-  "presets": ["es2015"],
-  "plugins": ["transform-object-assign"]
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": {
+        "browsers": ["last 2 versions", "IE 10"]
```
Comment (seanparmelee) [inline .babelrc]: this change was already in the branch. not sure if you prefer to keep it, but fwiw: https://jamie.build/last-2-versions

File: lib/MethodFactory.js [hunk 2]
```diff
@@ -107,11 +107,11 @@ module.exports = class MethodFactory {
       throw new TypeError('loglevelnext: Logger is undefined or invalid. Please specify a valid Logger instance.');
     }
 
-    for (const methodName of this.methods) {
+    this.methods.forEach((methodName) => {
```
Comment (seanparmelee) [inline lib/MethodFactory.js]: The polyfill didn't place nicely with the Babel transformation code that results from the `for..of` so I changed this to a `forEach`. IE 11 was thowing an `Object expected` error.

**Comment by shellscape** [comment]
Thanks for putting that together. I'll take a look at it later tonight.

**Comment by MatTheCat** [comment]
Using this branch I no longer have an error regarding `Symbol`. There is another problem affecting IE and Edge but it seems to be caused by webpack-hot-client.

**Comment by shellscape** [comment]
Thanks for your work on this 🍺 

In the future please keep from adding `dist` files and `package-lock`.

**Pull request merged**